channels/ht.m3u

```diff
@@ -1,7 +1,33 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-country="HT" tvg-language="" tvg-logo="" group-title="General",Tele Haiti (1088p) [Offline]
-http://66.175.238.147:1935/live/myStream/chunklist_w1749202216.m3u8
-#EXTINF:-1 tvg-id="TeleLouange.ht" tvg-country="HT" tvg-language="French" tvg-logo="https://i.imgur.com/vZmlq9N.jpg" group-title="",Tele Louange (720p)
-https://5790d294af2dc.streamlock.net/8124/8124/chunklist.m3u8
-#EXTINF:-1 tvg-id="TeleHaiti.ht" tvg-country="HT" tvg-language="French" tvg-logo="https://www.newsfromhaiti.com/images/TeleHaiti.jpg" group-title="General",TeleHaiti (1088p) [Offline]
+#EXTINF:-1 tvg-id="HaitiViralNews.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/eaPFJJs.jpg" group-title="",Haiti Viral News (720p) [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCcEY2-g-EEefxyYa1gtrk4g/live
+#EXTINF:-1 tvg-id="KajouTV.ht" tvg-country="US;HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/5txJUZx.jpg" group-title="",Kajou TV (720p)
+https://video1.getstreamhosting.com:1936/8055/8055/playlist.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/OJbRkQ6.jpg" group-title="",NETALKOLE TV (180p)
+https://watch.haitilive.net/stream/netalkole/public/tv/index.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://uni01rtmp.tulix.tv/4vehtv/4vehtv-firetv/playlist.m3u8
```

Comment:
I'll aprove it as is for now but tulix.tv can have it's separate playlist file @freearhey